modules/browser/BrowserCapabilities.js

```diff
@@ -276,40 +287,14 @@ export default class BrowserCapabilities extends BrowserDetection {
         return false;
     }
 
-    /**
-     * Returns whether or not the current browser should be using the new
-     * getUserMedia flow, which utilizes the adapter shim. This method should
-     * be temporary and used while migrating all browsers to use adapter and
-     * the new getUserMedia.
-     *
-     * @returns {boolean}
-     */
-    usesNewGumFlow() {
-        const REQUIRED_CHROME_VERSION = 61;
-
-        if (this.isChrome()) {
-            return !this.isVersionLessThan(REQUIRED_CHROME_VERSION);
-        }
-
-        if (this.isFirefox() || this.isSafariWithWebrtc()) {
-            return true;
-        }
-
-        if (this.isChromiumBased()) {
-            return this._getChromiumBasedVersion() >= REQUIRED_CHROME_VERSION;
-        }
-
-        return false;
-    }
-
     /**
      * Checks if the browser uses webrtc-adapter. All browsers using the new
      * getUserMedia flow and Edge.
      *
      * @returns {boolean}
      */
     usesAdapter() {
-        return this.usesNewGumFlow() || this.isEdge();
+        return this.isSupported() || this.isEdge();
```

Comment:
This is not correct. The adapter doesn't really support RN, so we shouldn't use it there. Also, I think we can now drop the Edge check, since it's not supported. Edgium is, but that's already covered by the previous check. I think this should read `return !this.isReactNative();`

Reply:
Ok, didn't know that adapter doesn't support RN, also I wasn't sure why Edge was here when it is not supported.